src/Calendar/Calendar.driver.js

```diff
@@ -44,6 +46,7 @@ const calendarDriverFactory = ({element, wrapper}) => {
     pressLeftArrow: () => ReactTestUtils.Simulate.keyDown(getFocusedDay(), {key: 'ArrowLeft', keyCode: 37}),
     pressRightArrow: () => ReactTestUtils.Simulate.keyDown(getFocusedDay(), {key: 'ArrowRight', keyCode: 39}),
     getSelectedDay: () => getSelectedDay().textContent,
+    getSelectedDays: () => map(getSelectedDays(), 'textContent'),
```

Comment:
please use `Array.prototype.map` with `Array.from` instead of lodash dep.

```suggestion
    getSelectedDays: () => Array.from(getSelectedDays()).map(({textContent}) => textContent)
```

Reply:
I agree

Reply:
@argshook BW are there any lodash helpers that you DO think are worth using?
Can you make a list? Add it to the Contribution guide.

Reply:
i don't think we need to blacklist anything, what will we do in reviews then? :)

lodash does have useful things, like `zip`, `difference`, `intersection`, `debounce`, `throttle`, `uniq`, `flow` even `pick`, `omit`. it has many tools but most of the time i see `map`, `filter` or something like `isUndefined` being used.